lib/ngx/ssl.lua

```diff
@@ -251,6 +261,25 @@ function _M.raw_server_addr()
 end
 
 
+function _M.server_port()
+    local r = get_request()
+    if not r then
+        error("no request found")
+    end
+
+    local rc = ngx_lua_ffi_ssl_server_port(r, uintp, errmsg)
+    if rc == FFI_OK then
+        return uintp[0]
+    end
+
+    if rc == FFI_DECLINED then
```

Comment:
The C code doesn't return `FFI_DECLINED`.

Reply:
update